python/ml4ir/base/model/calibrations/temperature_scaling.py

```diff
@@ -0,0 +1,247 @@
+"""
+Implementing temperature scaling proposed in https://github.com/gpleiss/temperature_scaling
+The tensorflow implementation and functions are inspired by
+https://www.tensorflow.org/probability/examples/Optimizers_in_TensorFlow_Probability
+"""
+import sys
```

Comment:
let's call the dir `calibration` instead of `calibrations`
PITA, but I strongly believe it makes more sense.

Reply:
Haha, yes. I was going to say this as well, but avoided because everything else had an `s` at the end. I like `calibration` better too.